geoips/filenames/base_paths.py

```diff
@@ -275,6 +281,11 @@ def initialize_paths():
         paths["OUTPUT_CHECKER_THRESHOLD_IMAGE"]
     )
 
+    # Casting some values to bool
```

Comment:
@jsolbrig this addresses your well-founded concerns about the bool casting of the DONT_AUTOGEN_GEOLOCATION var